lang/py/src/avro/datafile.py

```diff
@@ -16,6 +16,7 @@
 """
 Read/Write Avro File Object Containers.
 """
+import bz2
```

Comment:
Do we want to consider supporting using avro on systems that don't have bzip headers? https://stackoverflow.com/questions/12806122/missing-python-bz2-module

We could make it optional at install time and use the same if/then approach as we do with snappy. Maybe we should make this our standard implementation for all compression algos?